lightning/src/util/events.rs

```diff
@@ -182,6 +182,12 @@ pub enum HTLCDestination {
 		/// Short channel id we are requesting to forward an HTLC to.
 		requested_forward_scid: u64,
 	},
+	/// We couldn't forward to the outgoing scid. An example would be attempting to send a duplicate
+	/// intercept HTLC.
+	InvalidForward {
+		/// Short channel id we are requesting to forward an HTLC to.
```

Comment:
nit: "Short channel id we are requested to forward an HTLC to" or dunno if it's my English blocking here but we're requested by the HTLC sender in knowledge of the routing hint to do a JIT channel. Also it can precise for now it's always a fake scid.

Reply:
Saving for followup since this is going to turn from a u64 into an `InterceptNextHop`, see that enum's docs. So it should be pretty clear that it's fake